EnglishFilter/sections/foreign.txt

```diff
@@ -9133,6 +9133,8 @@ topsinhalamp3.com#?##sitbl > tbody > tr[-ext-has='a[href^="https://ylx-4.com"]']
 !------------------------------ Thai ---------------------------------
 !---------------------------------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/117754
+teroradio.com,becteroradio.com##body #head-1
```

Comment:
It block login buttons
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/8361299/167197382-9e1bfc3a-b053-4d30-9751-a7bbf96e765c.png)

</details>

```
teroradio.com#$##leaderboard-billboard { visibility: hidden !important; height: 0 !important; }
teroradio.com###leaderboard
```